imod/tests/test_mf6/test_multimodel/test_mf6_partitioning_unstructured.py

```diff
@@ -338,20 +338,32 @@ def test_partitioning_unstructured_with_well(
 @parametrize_with_cases("partition_array", cases=PartitionArrayCases)
 def test_partition_transport(    tmp_path: Path,
     circle_model_transport: Modflow6Simulation,partition_array: xu.UgridDataArray):
-
     circle_model_transport.write(tmp_path)
     circle_model_transport.run()
     concentration = circle_model_transport.open_concentration()
+    budget = circle_model_transport.open_transport_budget(["salinity"])
 
     new_circle_model = circle_model_transport.split(partition_array)
     new_circle_model.write(tmp_path/"split", binary=False)
     new_circle_model.run()       
     new_concentration = new_circle_model.open_concentration()
+
     new_concentration = new_concentration.ugrid.reindex_like(concentration)
+    new_budget = new_circle_model.open_transport_budget(["salinity"])
+    new_budget = new_budget.ugrid.reindex_like(budget)
     np.testing.assert_allclose(
         concentration.values, new_concentration["concentration"].values, rtol=7e-5, atol=3e-3
     )
 
+    # Compare the budgets before and after splitting the simulation. 
+    # The tolerance is set as the maximum value in the  gwt-gwt 
+    # exchange as this contains mass transport in the split case that would 
+    # be in one of the other arrays in the unsplit case.  
```

Comment:
Can we account for this in the test in a more informed way? In this state, we cannot track if Modflow6 behavior alters, which is good to know.

For example:

```python
exchange_is_in_budget = new_budget["gwt-gwt"].notnull() & budget[budget_term].notnull()
exchange_new_budget = new_budget["gwt-gwt"].where(exchange_is_in_budget)

actual_budget = new_budget[budget_term].where(~exchange_is_in_budget, exchange_new_budget)

```

This doesn't work if two budget terms share an exchange cell, but for this test that is not necessary: We can always modify the test case to prevent budget terms sharing an exchange cell. For example, make it single layer to make flow-lower-face all zero. No SSM in exchange cell. etc.

Reply:
done, now skips the cells on the exchange boundary when comparing budget results